scripts/run-alphanet-parachain.sh

```diff
@@ -77,16 +77,22 @@ $PARACHAIN_BINARY \
   --rpc-port $((PARACHAIN_PORT + 10 + 1)) \
   --ws-port $((PARACHAIN_PORT + 10 + 2)) \
   --validator \
+  --rpc-cors all \
+  --rpc-methods=unsafe \
+  --execution wasm \
+  --wasm-execution compiled \
   --name parachain_$PARACHAIN_INDEX \
   $PARACHAIN_BASE_PATH \
-  '-linfo,evm=trace,ethereum=trace,rpc=trace' \
-  --chain $PARACHAIN_SPEC_PLAIN  \
+  '-linfo,evm=debug,ethereum=trace,rpc=trace,cumulus_collator=debug,txpool=debug' \
+  --alice  \
+  --chain staging \
+  --account-id 6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b \
   $PARACHAIN_BOOTNODES_ARGS \
   -- \
     --node-key ${PARACHAIN_KEYS[$PARACHAIN_INDEX]} \
+    --chain rococo-local \
```

Comment:
@JoshOrndorff Is it the right way to specify the embedded relay chain ?

Reply:
For posterity: Alan and had a call and realized that Polkadot and Moonbeam generate the relay spec differently even from the same codebase and same physical machine. So we will keep the `generate-relay-specs.sh` script in tact to generate a raw spec.

Reply:
This is mostly due because we are running an already built-in polkadot binary (through docker, to avoid having to compile polkadot).
Compiling polkadot and using that binary will probably generate the same raw specs as moonbase binary